tests/dining/test_forms.py

```diff
@@ -0,0 +1,48 @@
+from datetime import time, date
+
+from django.test import TestCase
+
+from Dining.forms import CreateSlotForm
+from UserDetails.models import Association, User, UserMemberships
+
+
+class CreateSlotFormTestCase(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        """
+        Do not modify the objects created here in tests, see:
+        https://docs.djangoproject.com/en/2.1/topics/testing/tools/#django.test.TestCase.setUpTestData
+        """
+        cls.association1 = Association.objects.create(name="Auletes")
+        cls.association2 = Association.objects.create(name="Ensuite")
```

Comment:
No offense, maar we gaan niet testen met SUBVERENIGINGEN VAN QUADRIVIUM.
Doe dan gewoon Quadrivium en Knights ofzo. @mhvis 
:P